File: src/main/java/com/hris/HRIS/model/TaxModel.java
package com.hris.HRIS.model;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Getter
@Setter
@ToString

@Document(collection = "tax")
public class TaxModel {
    @Id
    private String id;
    private String organizationId;
    private double min;
    private double max;
    private double rate;
}

File: client/src/app/shared/data-models/tax.model.ts
import { Injectable } from "@angular/core";

@Injectable({
    providedIn: 'root'

})
export class TaxModel{

    id: String = "";
    organizationId: String = "";
    min: number = 0.0;
    max: number = 0.0;
    rate: number = 0.0;

}